packages/graphql/src/translate/queryAST/factory/OperationFactory.ts

```diff
@@ -516,6 +540,21 @@ export class OperationsFactory {
         }
     }
 
+    private getParsedProjectionFields(
```

Comment:
I'm not 100% sure about this abstraction, If we keep it I would reflect the fact that it parses only aggregation projections!